client/components/Entry/EditEntryForm.vue

```diff
@@ -14,19 +14,18 @@ export default {
     return {
       url: `/api/entries/${this.entryId}`,
       method: 'PATCH',
-      hasBody: true,
       title: 'Edit entry',
-      callback: () => {
-        this.$router.go(-1);
-        this.$store.dispatch('refreshEntries');
+      callback: async () => {
         this.$store.commit('alert', {
           message: 'You\'ve saved changes to the entry!', status: 'success'
         });
+        await this.$store.dispatch('refreshEntries');
+        this.$router.go(-1);
```

Comment:
I like this change, though I wonder if it's worth adding a progress bar during the refresh to give better indication that it's still loading and not just being non-responsive?

Reply:
thats a good thought. I like that it doesn't flash anymore, but it does require two round trips. I'll think about this some more. One alternative would be to update the store immediately on the front end rather than waiting for the refresh. (could still have the refresh as a backup to ensure consistency if we want)